compiler/ballerina-backend-jvm-old/src/main/ballerina/src/compiler_backend_jvm/jvm_desugar_phase.bal

```diff
@@ -136,6 +137,10 @@ function updateParamTypesWithDefaultableBooleanVar(bir:BType?[] funcParams) retu
         index += 2;
         counter += 1;
     }
+    if (!(restType is ())) {
```

Comment:
I feel it would increase readability if you use restType is bir:BType instead of the negated statement? WDYT
```suggestion
    if ( restType is bir:BType ) {
```

Reply:
Please see above comment.